Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Build the Ruby image from source
+FROM ruby:3.0 AS ruby-build
```

Comment:
Can you contribute the Dockerfile under a different PR?

Reply:
That wasn't supposed to be here...  
I loathe git

Reply:
Removed from this PR.